packages/resolver.go

```diff
@@ -7,7 +7,7 @@ package packages
 import "net/http"
 
 type RemoteResolver interface {
-	RedirectArtifactsHandler(w http.ResponseWriter, r *http.Request, p *Package)
-	RedirectStaticHandler(w http.ResponseWriter, r *http.Request, p *Package, resourcePath string)
-	RedirectSignaturesHandler(w http.ResponseWriter, r *http.Request, p *Package)
+	ForwardArtifactsHandler(w http.ResponseWriter, r *http.Request, p *Package)
+	ForwardStaticHandler(w http.ResponseWriter, r *http.Request, p *Package, resourcePath string)
+	ForwardSignaturesHandler(w http.ResponseWriter, r *http.Request, p *Package)
```

Comment:
Perhaps we can remove the verb from here, and we could have different implementations, one with redirect, another one with proxy forwarding...
```suggestion
	ArtifactsHandler(w http.ResponseWriter, r *http.Request, p *Package)
	StaticHandler(w http.ResponseWriter, r *http.Request, p *Package, resourcePath string)
	SignaturesHandler(w http.ResponseWriter, r *http.Request, p *Package)
```

Reply:
That makes totally sense, let's remove the verb